prettier.el

```diff
@@ -7,7 +7,7 @@
 ;; Created: 7 Nov 2018
 ;; Keywords: convenience, languages, files
 ;; Homepage: https://github.com/jscheid/prettier.el
-;; Package-Requires: ((emacs "26.1") (iter2 "0.9") (nvm "0.2") (editorconfig "0.8"))
+;; Package-Requires: ((emacs "27.1") (iter2 "0.9") (nvm "0.2") (editorconfig "0.8"))
```

Comment:
Hi @asbish, thanks for this!

I'm not sure about bumping up the minimum version here. Even if we're no longer testing against it, we're still compatible for now so I think better to leave as-is.

In case we break compatibility with 26 in the future it won't be a big deal, people can just open a bug report in that case.

Reply:
Oh, indeed. I had not noticed that point. 
I'll revert it back to the original.